internal/namespaces/instance/v1/custom_server_create.go

```diff
@@ -253,17 +256,20 @@ func instanceServerCreateRun(ctx context.Context, argsI interface{}) (i interfac
 	// More format details in buildVolumeTemplate function.
 	//
 	if len(args.AdditionalVolumes) > 0 || args.RootVolume != "" {
-		// Get default organization ID.
-		organizationID := args.OrganizationID
-		if organizationID == nil {
-			orgFromCtx := core.GetOrganizationIDFromContext(ctx)
-			if orgFromCtx != "" {
-				organizationID = &orgFromCtx
+		// Get default project ID.
+		projectID := args.ProjectID
```

Comment:
Project and organization **should not be mixed**.

The same check done for organizationID should be done on projectID independantly, then `buildVolumes` should take both as parameters and pass them in the SDK.